cljfmt/src/cljfmt/core.cljc

```diff
@@ -280,10 +280,22 @@
     (or (symbol-matches-key? (fully-qualified-symbol possible-sym context) key)
         (symbol-matches-key? (remove-namespace possible-sym) key))))
 
+(defn- parent-is-thread-first? [zloc context]
+  (some #(form-matches-key? zloc % context) #{'-> 'as-> 'some-> 'cond->}))
+
+(defn- is-first-parameter? [zloc]
+  (= (z/right (z/leftmost zloc)) zloc))
```

Comment:
In Clojure, by convention we omit the `is` from predicates, and use "argument" over "parameter". So, `first-argument?` would be a more idiomatic name.